src/realm/sync/client.cpp

```diff
@@ -1701,28 +1708,28 @@ void SessionWrapper::report_progress(ReportedProgress& p, DownloadableProgress d
         upload_estimate = calculate_progress(p.uploaded, p.uploadable, m_final_uploaded);
 
     bool download_completed = p.downloaded == 0;
-    double download_estimate = 1.00;
+    p.download_estimate = 1.00;
     if (m_flx_pending_bootstrap_store) {
         if (m_flx_pending_bootstrap_store->has_pending()) {
-            download_estimate = downloadable.as_estimate();
+            p.download_estimate = downloadable.as_estimate();
```

Comment:
I think if you moved this down to line 1716, right before download_completed gets set, that would solve the issue @kiburtse is calling out above.

Reply:
Right, obviously we can get download progress estimates while in the steady state and not just when bootstrapping.